packages/dev/core/src/Cameras/Inputs/freeCameraDeviceOrientationInput.ts

```diff
@@ -222,6 +227,14 @@ export class FreeCameraDeviceOrientationInput implements ICameraInput<FreeCamera
     public getSimpleName(): string {
         return "deviceOrientation";
     }
+
+    /**
+     * Set the smoothFactor for devive orientation sensors, to limit objects shaking
+     * From AR.js documentation, best value should be 0.9
+     */
+     public setSmoothFactor(smoothFactor=0.9): void {
+        this._smoothFactor = smoothFactor
+    }

```

Comment:
any reason why we don't want to provide a getter (or avoid getting/setting altogether)? A public member will be fine. Otherwise - there i no reason to provide a default value in a setter.

Reply:
every property in the class was private, so I assumed it was a design choice, but I also think we are adults (my python background ;-) ). so let's make it public indeed.

Reply:
"adults"